tools/depmod.c

```diff
@@ -1572,6 +1582,25 @@ static int depmod_load_modules(struct depmod *depmod)
 
 load_info:
 		kmod_module_get_info(mod->kmod, &mod->info_list);
+		kmod_list_foreach(l, mod->info_list) {
+			const char *key = kmod_module_info_get_key(l);
+
+			if (streq(key, "alias")) {
+				const char *value = kmod_module_info_get_value(l);
+				array_append(&mod->alias_values, value);
```

Comment:
These can return error values in OOM conditions and should be handled accordingly.